build.gradle.kts

```diff
@@ -469,6 +469,16 @@ tasks.register("pythonPreCommit") {
   dependsOn(":sdks:python:test-suites:tox:py311:preCommitPy311")
 }
 
+tasks.register("pythonNotebooksPreCommit") {
+    /* doLast {
+      exec {
+        executable("sh")
+        args ("-c", ". ${envdir}/bin/activate && python -m apache_beam.examples.wordcount --runner DirectRunner --output /tmp/output.txt")
+      }
+     }*/
+  dependsOn(":sdks:python:wordCount")
```

Comment:
This seems wrong - this should be depending on a task in `:examples:notebooks`, right?